src/SFA.DAS.EAS.Web/comt-assets/css/screen.css

```diff
@@ -3073,6 +3073,7 @@ a#link-settings:focus {
 
 .success-summary {
   border-color: #28a197;
+  text-align:center;
```

Comment:
The css files in the `comt-assets/css` directory are generated from the source sass files, so the change to `success-summary` also needs to be made in `src\sass\_notifications.scss`, otherwise your changes will be blatted the next time the css files are regenerated.

Reply:
Done